lib/src/Uncertainty/Distribution/openturns/MarshallOlkinCopula.hxx

```diff
@@ -0,0 +1,113 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The MarshallOlkinCopula distribution
+ *
+ *  Copyright 2005-2019 Airbus-EDF-IMACS-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef OPENTURNS_MARSHALLOLKINCOPULA_HXX
+#define OPENTURNS_MARSHALLOLKINCOPULA_HXX
+
+#include "openturns/CopulaImplementation.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+/**
+ * @class MarshallOlkinCopula
+ *
+ * The MarshallOlkinCopula distribution.
+ */
+class OT_API MarshallOlkinCopula
+  : public CopulaImplementation
+{
+  CLASSNAME
+public:
+
+  /** Default constructor */
+  MarshallOlkinCopula();
+
+  /** Parameters constructor */
+  explicit MarshallOlkinCopula(const Scalar alpha,
```

Comment:
no need for explicit keyword here : more than 1 required parameter

Reply:
I did not know. I'll fix this.